jb/itch5/order_book_depth.hpp

```diff
@@ -1,8 +1,9 @@
-#ifndef jb_itch5_order_book_hpp
-#define jb_itch5_order_book_hpp
+#ifndef jb_itch5_order_book_depth_hpp
```

Comment:
We decided to copy pasted order_book into order_book_depth. 
Then, we realized that order_book_depth was implementing both functionalities.
So I deleted order_book... anyway. I am going to go the other way around.